### PR TITLE
boards: doc: Add programming tabs for mimx8mq_evk

### PR DESCRIPTION
**Pull request by jeppenodgaard**
Improve documentation by putting programming methods into tabs.

### CODE REVIEW

**Review by dbaluta — DISMISSED** [review]

File: boards/arm/mimx8mq_evk/doc/index.rst [hunk 1]
```diff
@@ -140,48 +140,48 @@ The available configurations are:
...
+      .. code-block:: console
 
-From serial:
+         fatload mmc 0:1 0x40480000 zephyr.bin
+         cp.b 0x40480000 0x7e0000 0x8000
+         bootaux 0x7e0000
 
-This procedure requires `screen` and `lrzsz` to be installed.
+   .. group-tab:: From serial
+      This procedure requires `screen` and `lrzsz` to be installed.
```
Comment (kartben) [inline boards/arm/mimx8mq_evk/doc/index.rst:159]:
```suggestion
      This procedure requires ``screen`` and ``lrzsz`` to be installed.
```

File: boards/arm/mimx8mq_evk/doc/index.rst [hunk 2]
```diff
@@ -140,48 +140,48 @@ The available configurations are:
...
 
-.. code-block:: console
+      .. code-block:: console
 
-   loadx 7e0000 115200
+         loadx 7e0000 115200
 
-Send the compiled ``zephyr.bin`` with ``sx`` by pressing Ctrl-a followed by ':'
-and write:
+      Send the compiled ``zephyr.bin`` with ``sx`` by pressing Ctrl-a followed by ':'
```
Comment (kartben) [inline boards/arm/mimx8mq_evk/doc/index.rst:173]:
```suggestion
      Send the compiled ``zephyr.bin`` with ``sx`` by pressing :kbd:`Ctrl-a` followed by :kbd:`:`
```

**Review by kartben — CHANGES_REQUESTED** [review]
good idea!
Note that you may also want to look into fixing the rendering of L129-L139, it doesn't look quite right and probably shouldn't be a code block? Might also make sense to explain what are the differences between all the available flash and sram options?

**Comment by jeppenodgaard** [comment]
> good idea! Note that you may also want to look into fixing the rendering of L129-L139, it doesn't look quite right and probably shouldn't be a code block? Might also make sense to explain what are the differences between all the available flash and sram options?

Thanks @kartben ! I've added your suggestions.
I am not quite sure how to fix that part of the documentation.
It was added in https://github.com/zephyrproject-rtos/zephyr/pull/37158. Maybe @KwonTae-young has an idea?

**Comment by jeppenodgaard** [comment]
Hi @kartben.
Since @KwonTae-young has not yet responded, is it ok to consider your L129-L139 suggestions to be out of scope of this PR?

**Review by kartben — APPROVED** [review]

**Comment by jeppenodgaard** [comment]
Hi @dbaluta. Can you re-review please?

**Review by iuliana-prodan — APPROVED** [review]

**Review by jhedberg — APPROVED** [review]

**Review by fabiobaltieri — APPROVED** [review]

**Pull request merged**